index.html

```diff
@@ -8,198 +8,244 @@
     <meta charset="UTF-8">
```

Comment:
Please use linter config for html+css+js from [this](https://github.com/microverseinc/linters-config/tree/master/html-css-js) link.